templates/webapi/main/parent_group_overview.html.ep

```diff
@@ -15,27 +15,131 @@
         </button>
     </form>
 % }
-<h2>
+
+<script>
```

Comment:
This should go in a separate file and should be formatted in consistency with other files (4 space indentation, space before curly braces, ...).

Reply:
Yes, I agree